CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Change history for stripes-smart-components
 
-## [7.0.1] (IN PROGRESS)
+## [7.1.0] (IN PROGRESS)
```

Comment:
IMO it's better to remove the version at all - `## (IN PROGRESS)`. And put release version only during new release creation.

Reply:
in acq we keep it's always in progress at the top and add release version when do releases

https://github.com/folio-org/ui-orders/blob/master/CHANGELOG.md#in-progress